lib_static/open_project/authentication/strategies/warden/jwt_oidc.rb

```diff
@@ -30,7 +30,7 @@ def authenticate!
                                            error_description: "Requires scope #{scope} to access this resource."
                 end
 
-                user = User.find_by(identity_url: "#{provider.slug}:#{payload['sub']}")
+                user = UserAuthProviderLink.where(auth_provider: provider, external_id: payload["sub"]).first&.user
```

Comment:
:green_circle: 

```suggestion
                user = UserAuthProviderLink.find_by(auth_provider: provider, external_id: payload["sub"])&.user
```

Or maybe even better:

```suggestion
                user = provider.user_auth_provider_links.find_by(external_id: payload["sub"])&.user
```